providers/fathom/support.go

```diff
@@ -12,13 +12,20 @@ func supportedOperations() components.EndpointRegistryInput {
 	readSupport := []string{
 		"meetings", "teams", "team_members",
 	}
+	writeSupport := []string{
```

Comment:
are there any other objects that are available for writing, or is it just the one? If there are limitations, can yoi please document them here?